### DISCUSSION TITLE
Please, work with advanced answer sounds, it works very veery good with sounds - this is a suggestion

### DISCUSSION THREAD

**Discussion by Jota-veh** [Unlucky-Life/ankimon#143] [Announcements]
hey man, look at a addon called advanced answer sounds, it works waaaay better than audivisual-feedback. works very very good with .wav files. and i see no bugs with it. also, it has ambient sounds option, if you intend to make this "game enviroment feeling sounds". just a suggestion!

https://ankiweb.net/shared/info/1167194350

**Comment by Unlucky-Life** [comment]
@Jota-veh Hey man thanks for the great recommendation, I will check it out !
Btw love the Forrest Gump Profile picture, one of my favorite movies!
I definetly thought about adding ambient sounds, so this would be perfect !

**Comment by Unlucky-Life** [comment]
@Jota-veh 
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/564d65d6-2f68-4ce3-9f49-0340bd5bd13f)
Currently im thinking about something like this ^ Im thinking about integrating a button to play and pause ambient music

**Comment by Unlucky-Life** [comment]
Do you know if you had any sound issues with java script ? 
![image](https://github.com/Unlucky-Life/ankimon/assets/77027147/4753fd57-7430-4646-b11d-36964aaecdcb)

**Reply by Jota-veh** [reply]
well, I only use sounds with .wav and only sounds I use, i've downloaded and put on  advanced answers sounds folder. IDK about these languages, but i think you could see on the addon itself. But, that app audiovisual feedback wasn't working for me..., even when i converted those audio files to .wav 

anyway, i like these ideia. because i think you'll implement some "damage" taken and damage cause on the battle, and these sounds doesn't occur everytime you hit the answer button, i think, like you did on the addon, on the battle turn, for me the turn is two cards, then I take or cause damage, or both, after answered these two cards, right ? so those feedback audios about damage and gain xp, etc, will be veeeeery motivating, for sure.   

btw, if you like forrest gump, then we had some taste in common for audiovisual, maybe? because it's one of my favorite movie as well
so....
i'm using some snes audios for anki. take a look at them  , idk, maybe for inspiration, at least?

also, i think i'll share with you my answer sounds. i like those old videogames sounds because they are not distracting you, and they are not becoming boring after a very long time - because they thought about that when they composed them.


https://drive.google.com/drive/folders/1s6J732BD945n5zPxabHR46ZdVdij7bXb?usp=sharing

**Reply by Unlucky-Life** [reply]
@Jota-veh the damage sounds are based on the damage you have done => multiplier greater than 1 "very effective damage sound2; greater than 1.5 => "super effective damage sound"; lower than 1 => "uneffective damage sound"; 1 = "effective damage"

So its not directly corresponding to pressing good, again etc.

You said you dont here the sounds, have you turned on the sounds in the settings ? 
=> "sounds" = true in the Ankimon Addon manager settings to allow the sounds to be active
if you would like pokemon battle sounds you would also need to activate them in the settings by setting "battle_sounds" to true

**Reply by Unlucky-Life** [reply]
@Jota-veh Yes, I think we can find a common here ! 
Ill check it out (its a huge addon though for only sounds).

Could you explain a bit more how the sounds didnt work for you ?
(Like what Anki Version and System you are using at the moment)

**Reply by Unlucky-Life** [reply]
@Jota-veh i received the Resources, so you can take them down ! (Thanks for the resources, Ill take a look at it !)

**Reply by Jota-veh** [reply]
Anki 23.12.1 (1a1d4d54)  (ao)
Python 3.9.15 Qt 6.6.1 PyQt 6.6.1
Platform: Linux-6.6.32-1-MANJARO-x86_64-with-glibc2.39
kernel: 6.6.32-1 LTS

about sounds: i'm not getting any sounds for ankimon. idk if it happens under linux... since most anki users are mac/windows users.
and yes, i've tried with "true" option for game sounds and battle sounds. but sometimes i've got an error, and not sound at all everytime i tried.
but with that ankiarcade, on the other hand, I can play these sound effects (battle effects) pretty good, and plus, still using those answer sounds (advanced answer sounds) alongside at same time. but, i've deactivate those sound ambient from ankiarcade, also "deactivate" the sound effects sounds, because I was getting an error, sometimes. but on the volume i set the maximum for sound effects (ankiarcade) and everything is working fine (sword sound etc).   
![image](https://github.com/Unlucky-Life/ankimon/assets/96387624/a6225cfb-0e4b-448e-8a06-bdd3d2903ade)


only saying that, because, idk, i think maybe you can compare the three addons and find out what's going on.

but, hey man. i'm good, like a said, all of this is just suggestions! ^^
